packages/testsweets/example/lib/ui/main/main_view.dart

```diff
@@ -12,6 +12,12 @@ class MainView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ViewModelBuilder<MainViewModel>.reactive(
+      onModelReady: (model) {
+        TestSweetsNavigatorObserver.instance.setBottomNavIndex(
```

Comment:
This should move to the `getViewForIndex` function and we can set the index to the currentIndex being passed in.